clj-refactor.el

```diff
@@ -2042,8 +2047,11 @@ is not set to `:prompt'."
       (gethash :cljs aliases))))
 
 (defun cljr--js-alias-p (alias)
-  (and (cljr--cljs-file-p)
-       (string-equal "js" alias)))
+  (and (string-equal "js" alias)
+       (member "cljs" (condition-case nil
```

Comment:
I wonder if it would be clearer to only call member on the result of `(cljr--language-context-at-point)`, and just return t for the error/cljr--cljs-file-p case, instead of adding a possible list structure on the error branch to check with member if it falls through from the context-at-point case. Or is that problematic as the member call might trigger the error?

Alternatively, could we re-order this and do something like 

```
(and (string-equal "js" alias) 
         (or (cljr--cljs-file-p)
               (member (condition-case ... (error nil)))))
```
?

language-context-at-point *should* cover the case either way, but I wonder if shortcutting on the current cljs file first, and then falling into language-context-at point if we aren't certain it's cljs?

I may just be reacting to the potential error case in general though. Maybe we could wrap the body of cljr--language-context-at-point with a condition-case so it can never error and avoid handling edge cases in all of it's callers?